### PR TITLE
New Swagger API docs site

### PR DESCRIPTION
**Pull request by cmark**
This PR improves the basic Swagger UI to support navigation between the various APIs supported by Snow Owl.

### CODE REVIEW

File: core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/App.js [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+              position: 'fixed',
+              left: 0,
+            }}
+          >
+            <div className="logo">
+              <img src={`${process.env.PUBLIC_URL}/logo-snow-owl.png`} alt="logo-snow-owl" height="60" style={{maxWidth:"100%"}} />
+            </div>
+            <Menu
+              onSelect = { this.onMenuSelect }
+              defaultOpenKeys={['sub1']}
```
Comment (apeteri) [inline core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/App.js]: It looks like this property is only needed if at least one nested `SubMenu` is present.

File: releng/com.b2international.snowowl.server.update/.launch/mvn build (ext).launch [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<launchConfiguration type="org.eclipse.m2e.Maven2LaunchConfigurationType">
+    <booleanAttribute key="M2_DEBUG_OUTPUT" value="false"/>
+    <stringAttribute key="M2_GOALS" value="clean verify"/>
+    <booleanAttribute key="M2_NON_RECURSIVE" value="false"/>
+    <booleanAttribute key="M2_OFFLINE" value="false"/>
+    <stringAttribute key="M2_PROFILES" value=""/>
+    <listAttribute key="M2_PROPERTIES">
+        <listEntry value="nexus.url=http://nexus:8081/nexus"/>
```
Comment (apeteri) [inline releng/com.b2international.snowowl.server.update/.launch/mvn build (ext).launch]: This should be moved/rely upon `settings.xml` properties instead.

**Review by apeteri — CHANGES_REQUESTED** [review]

File: core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/logo.svg [hunk 3]
```diff
@@ -0,0 +1 @@
....23-.12s-3.58,0-5.33.12C88.45,50.57,90.2,48.48,92,46.56Zm-22.18-12A4.82,4.82,0,0,1,72.29,34a13.11,13.11,0,0,1,5.19,1.31,39.07,39.07,0,0,1,10,6.78l1.17,1.07a98.49,98.49,0,0,0-8.16,10,98.88,98.88,0,0,0-12.65,2.06c-.44-1.94-.8-3.84-1-5.67C65.8,42,67,36.24,69.81,34.6ZM64.53,85.26a58.75,58.75,0,0,1-5.54-2c-7.1-3-11.5-6.85-11.5-10.14S51.91,66,59,63.05a57.63,57.63,0,0,1,5.44-1.94A97.93,97.93,0,0,0,69,73.25,98.72,98.72,0,0,0,64.53,85.26Zm23,19.1c-8.19,7.33-13.31,8.11-15.14,8.11a4.69,4.69,0,0,1-2.42-.58c-2.86-1.63-4-7.38-3.09-15a56.07,56.07,0,0,1,1-5.78,99.51,99.51,0,0,0,12.58,2.06,97.17,97.17,0,0,0,8.24,10.08Zm4.57-4.51c-1.8-1.92-3.57-4-5.31-6.23,1.72.08,3.47.12,5.24.12s3.58,0,5.33-.11C95.63,95.85,93.87,97.93,92.09,99.85Zm22.18,12a4.82,4.82,0,0,1-2.48.59c-1.82,0-7-.8-15.16-8.1l-1.17-1.07a98.44,98.44,0,0,0,8.15-10,97,97,0,0,0,12.66-2.06c.44,1.94.79,3.84,1,5.67C118.27,104.42,117.12,110.18,114.27,111.81Zm10.8-28.44c-1.71.7-3.52,1.35-5.44,1.93a98.54,98.54,0,0,0-4.57-12.14,98.1,98.1,0,0,0,4.49-12,58.75,58.75,0,0,1,5.54,2c7.09,3,11.5,6.85,11.49,10.14S132.17,80.42,125.07,83.37ZM92,82.39a9.18,9.18,0,1,0-9.17-9.19A9.17,9.17,0,0,0,92,82.39Zm-69.32,54.5V26.2L17.89,31V141.66H139.94l4.78-4.77Z"/></svg>
```
Comment (apeteri) [inline core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/logo.svg]: Do we need the React nuclear logo in SVG form? 😄

Comment (apeteri) [reply core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/logo.svg]: image

Comment (cmark) [reply core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/logo.svg]: No, we do not. We need our own custom logo and favico. I'll remove/replace them. 😄

File: releng/com.b2international.snowowl.server.update/.launch/mvn build (ext).launch (hunk 2 above)
Comment (cmark) [reply releng/com.b2international.snowowl.server.update/.launch/mvn build (ext).launch]: Hmm, that file should be removed as well 😄

File: core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/App.js (hunk 1 above)
Comment (cmark) [reply core/com.b2international.snowowl.core.rest/snow-owl-api-docs/src/App.js]: Yeah, that should be removed as well.

**Review by apeteri — APPROVED** [review]
👍

**Pull request merged**